File: easybuy-manager-web/src/main/webapp/js/service/brandService.js
//品牌服务层
app.service('brandService',function($http){
    //读取列表数据绑定到表单中
    this.findAll=function(){
        return $http.get('../brand/findAll.do');
    }

    this.findPage=function (page,rows) {
      return   $http.get('../brand/findPage.do?page='+page+'&rows='+rows);
    }

    this.findOne=function(id){
       return   $http.get('../brand/findOne.do?id='+id);

    }
});

File: easybuy-sellergoods-interface/src/main/java/com/easybuy/sellergoods/service/BrandService.java
package com.easybuy.sellergoods.service;

import com.easybuy.pojo.PageResult;
import com.easybuy.pojo.TbBrand;

import java.util.List;

/**
 * 品牌服务层接口
 * @author Administrator
 *
 */
public interface BrandService {
    /**
     * 返回全部列表
     * @return
     */
    public List<TbBrand> findAll();

    /**
     * 返回分页列表
     * @return
     */
    public PageResult findPage(int pageNum, int pageSize);
}